### ISSUE TITLE
[Bug/Feature]: "Active citation key formula" not editable, overrides citation key format

### ISSUE THREAD

**Issue by hypernorth** [retorquere/zotero-better-bibtex#2498]
### Debug log ID

Y6R4KKQZ-euc

### What happened?

Hi @retorquere,  Here is what I'm currently experiencing. The Citation Key Formula is set to `Auth.Etal2.fold + '-' + year` however there's a second field called "Active citation key formula" with value `(auth.lower + year)`. Any new resources added to the library adopts the "active" formula. I cannot edit the "active" formula.  Thanks in advance for advice or assistance on this.

**Comment by retorquere** [comment]
please try build 4203

**Comment by retorquere** [comment]
`Auth.Etal2.fold + '-' + year` is not a valid formula -- try `authEtal2.fold + '-' + year`

**Comment by retorquere** [comment]
The edit field should have gone orange, and if you hover over the field, it should tell you that `Auth is not a valid field`.

**Comment by hypernorth** [comment]
Thank you - the edit field was orange. I updated the values to "authEtal2.fold + '-' + year" as directed. As shown in attached capture the additional "Active citation key formula" field is no longer showing. I tested by adding a new Title and it auto-pinned using the correct cite key formula. 

![image](https://user-images.githubusercontent.com/131813437/234635320-0f601e51-e908-4234-baab-b5996f7fb1e7.png)
